t/python/t0013-job-list-inactive.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
+###############################################################
+# Copyright 2014 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+
+import os
+import errno
+import sys
+import json
+import unittest
+import datetime
+import time
+from glob import glob
+
+import yaml
+import six
+
+import flux
+from flux import job
+from flux.job import Jobspec, JobspecV1, ffi
+
+
+def __flux_size():
+    return 1
+
+
+class TestJob(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.fh = flux.Flux()
+
+    @classmethod
+    def submitJob(self):
+        compute_jobreq = JobspecV1.from_command(
+            command=["sleep", "0"], num_tasks=2, num_nodes=1, cores_per_task=1
+        )
+        compute_jobreq.cwd = os.getcwd()
+        compute_jobreq.environment = dict(os.environ)
+        flux.job.submit(self.fh, compute_jobreq, waitable=True)
+
+    @classmethod
+    def execRPC(self, rpc_handle):
+        try:
+            jobs = rpc_handle.get_jobs()
+            return jobs
+        except EnvironmentError as e:
+            print("{}: {}".format("rpc", e.strerror), file=sys.stderr)
+            sys.exit(1)
+
+    attrs = [
+        "userid",
+        "state",
+        "name",
+        "ntasks",
+        "t_submit",
+        "t_run",
+        "t_inactive",
+    ]
+
+    # should return an empty list if there are no inactive jobs
+    def test_00_list_inactive_expect_empty_list(self):
+        rpc_handle = flux.job.job_list_inactive(
+            self.fh, time.time() - 3600, 10, self.attrs
+        )
+
+        jobs = self.execRPC(rpc_handle)
+
+        self.assertEqual(len(jobs), 0)
+
+    # make sure one job is read from RPC
+    def test_01_list_inactive_success(self):
+        # submit a sleep 0 job
+        self.submitJob()
+
+        # wait for eventual consistency
+        jobs = []
+        while True:
+            rpc_handle = flux.job.job_list(
+                self.fh, 0, self.attrs, states=flux.constants.FLUX_JOB_INACTIVE
+            )
+            jobs = self.execRPC(rpc_handle)
+            if len(jobs) == 1:
+                break
```

Comment:
Yes, sounds like a good idea. Maybe call it something like `waitForConsistency()`? I hesitate to just call it `wait()` because that might be confusing to have to differentiate between the `wait()` RPC and our own "@classmethod" implementation.

Reply:
`waitForConsistency()` sounds good to me!  Also, might be good to add some comments similar to the ones I have in `t2204-job-info` about how `job.wait()` syncs on the job manager, gotta wait for `job-info` eventual consistency, etc.